### DISCUSSION TITLE
v6.13.0

### DISCUSSION THREAD

**Discussion by zkochan** [pnpm/pnpm#3669] [Show and tell]
### Minor Changes

-   New command added for running packages in a tempory environment: `pnpm dlx <command> ...` [#3652](https://github.com/pnpm/pnpm/pull/3652).

### Patch Changes

-   Link the package's own binaries before running its lifecycle scripts [#3662](https://github.com/pnpm/pnpm/pull/3662).
-   Resolve peer dependencies from the root of the workspace when adding a new dependency or updating [#3667](https://github.com/pnpm/pnpm/pull/3667).
-   Ignore empty shasum in entries in package metadata [#3666](https://github.com/pnpm/pnpm/pull/3666).
-   Throw a meaningful error if a package has invalid shasum in its metadata [#3666](https://github.com/pnpm/pnpm/pull/3666).
-   Add `run` to NO_SCRIPT error example [#3660](https://github.com/pnpm/pnpm/pull/3660).

<hr /><em>This discussion was created from the release <a href='https://github.com/pnpm/pnpm/releases/tag/v6.13.0'>v6.13.0</a>.</em>

**Comment by shiftgeist** [comment]
Is 6.13.0 intentionally released under the `next` tag?

**Reply by zkochan** [reply]
I will make it latest soon
